src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml

```diff
@@ -14,157 +14,26 @@
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.08" />
-	<SolidColorBrush x:Key="MaterialOnSecondaryPressedBrush"
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.1" />
-	<SolidColorBrush x:Key="MaterialOnSecondaryDraggedBrush"
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.25" />
-	
-	<!-- Background brushes -->
-	<SolidColorBrush x:Key="MaterialBackgroundBrush"
-					 Color="{ThemeResource MaterialBackgroundColor}" />
-	<SolidColorBrush x:Key="MaterialOnBackgroundBrush"
-					 Color="{ThemeResource MaterialOnBackgroundColor}" />
-	<SolidColorBrush x:Key="MaterialOnBackgroundLowBrush"
-					 Color="{ThemeResource MaterialOnBackgroundColor}"
-					 Opacity="0.38" />
-	<SolidColorBrush x:Key="MaterialOnBackgroundMediumBrush"
-					 Color="{ThemeResource MaterialOnBackgroundColor}"
-					 Opacity="0.6" />
-
-	<!-- Surface brushes -->
-	<SolidColorBrush x:Key="MaterialSurfaceBrush"
-					 Color="{ThemeResource MaterialSurfaceColor}" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceLowBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.38" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceMediumBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.75" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceHoverBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.04" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceFocusedBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.12" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceSelectedCBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.08" />
-	<SolidColorBrush x:Key="MaterialOnSurfacePressedBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.1" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceDraggedBrush"
-					 Color="{ThemeResource MaterialOnSurfaceColor}"
-					 Opacity="0.32" />
-
-	<!-- Error brushes -->
-	<SolidColorBrush x:Key="MaterialErrorBrush"
-					 Color="{ThemeResource MaterialErrorColor}" />
-	<SolidColorBrush x:Key="MaterialOnErrorBrush"
-					 Color="{ThemeResource MaterialOnErrorColor}" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceMedEmphasisBrush"
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.6" />
-	<SolidColorBrush x:Key="MaterialOnSurfaceDisabledBrush"
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.38" />
-
 	<!-- Success brushes -->
 	<SolidColorBrush x:Key="MaterialSuccessBrush"
 					 Color="{ThemeResource MaterialSuccessColor}" />
 	<SolidColorBrush x:Key="MaterialOnSuccessBrush"
 					 Color="{ThemeResource MaterialOnSuccessColor}" />
+	
+	<!-- ToggeSwitch Brushes -->
+
+	<!-- Toggle On -->
+	<StaticResource x:Key="MaterialToggleSwitchOnBackgroundBrush"
```

Comment:
I do not see the added toggle StaticResource used in this PR, still needed?

Reply:
Yes they are used in the material toggle switch style. It overrides some of the colors defined here : https://github.com/unoplatform/Uno.Themes/blob/86da421f6e91ffe4831356e41ab7f3ee55d6373b/src/library/Uno.Material/Styles/Controls/v1/ToggleSwitch.Base.xaml